charts/vault-secrets-operator/Chart.yaml

```diff
@@ -7,4 +7,4 @@ maintainers:
   - name: Rico Berger
     url: https://ricoberger.de
 name: vault-secrets-operator
-version: 1.2.1
+version: 1.2.2
```

Comment:
Can you revert this change please. The version bumping is automated via Makefile.